resources/wikia/Resources.php

```diff
@@ -183,6 +184,13 @@
 		],
 	],
 
+	'moment' => [
```

Comment:
I don't know the details of our AMD implementation, but I believe this just adds the library as a possible dependency to include in other JS files. It should only actually be loaded when required from another JS file.

Right now, since it was just added, it is only used by ext.embeddableDiscussions.js. That file is however loaded on any article page for communities that have discussions present.

Perhaps someone else has more knowledge of exactly how the AMD loader for Oasis works?